stablehlo/dialect/StablehloOps.cpp

```diff
@@ -2477,10 +2450,9 @@ ParseResult WhileOp::parse(OpAsmParser& parser, OperationState& result) {
     if (succeeded(parser.parseOptionalRParen())) break;
     if (failed(parser.parseComma())) return failure();
   } while (true);
-  if (!operands.empty()) {
+  if (!operands.empty())
     if (parser.parseColon() || parser.parseTypeList(result.types))
       return failure();
```

Comment:
```
// Use braces on the outer `if` to avoid a potential dangling `else`
// situation.
if (isa<VarDecl>(D)) {
  if (shouldProcessAttr(A))
    handleAttr(A);
}
```